pkg/virt-handler/vm.go

```diff
@@ -479,8 +479,9 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 
 		if len(domain.Spec.Devices.Interfaces) > 0 || len(domain.Status.Interfaces) > 0 {
 			// This calculates the vmi.Status.Interfaces based on the following data sets:
-			// - vmi.Status.Interfaces - previously calculated interfaces, this can contains data
-			//   set in the controller (pod IP) which can not be deleted, unless overridden by Qemu agent
+			// - vmi.Status.Interfaces - previously calculated interfaces, this can contains data (pod IP)
+			//   set in the previous loops when there are no interfaces which can not be deleted,
```

Comment:
I would change to something like this -
"this can contain data (pod IP) set in the previous loops (when there were no interfaces), which can not be deleted, ...".

Currently it hard to understand what cannot be deleted. the interfaces? the pod IP?

Reply:
Corrected